src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -1729,10 +1740,14 @@ public Response getSiteMigrationInBulkUpload(
 			// now that we get the site ids for batch upload
 			// start the batch process
 			String bulkMigrationId = java.util.UUID.randomUUID().toString();
-
+			
 			for (String siteId : bulkUploadSiteIds) {
 				// for each site id, start the migration process
 				// associate it with the bulk id
+				
+				// add admin user to site with Owner role
```

Comment:
How do we guarantee that the admin is removed even if things go wrong?  Does it make sense to use a try/finally block?